.github/workflows/dotnet.yml

```
@@ -99,8 +163,7 @@ jobs:
         id: deploy-to-webapp
         uses: azure/webapps-deploy@v2
         with:
-          app-name: 'housing-repairs-online-api-newark'
-          slot-name: 'Production'
+          app-name: "${{secrets.SERVICE_NAME}}-staging"
```

I don't think this should this have the `-staging` - is it required?

```suggestion
          app-name: "${{secrets.SERVICE_NAME}}"
```

---

Shouldn't this be the name of the app service slot? @Bhavesh-Patel